packages/client/src/createTRPCUntypedClient.ts

```diff
@@ -1,4 +1,4 @@
-import { AnyRouter } from '@trpc/server';
+import { AnyRouter } from '@trpc/server/unstableInternalsExport';
```

Comment:
Should be public imo

Reply:
Should we rename it to `TRPCRouter`?

Reply:
1. rename it
2. deprecate and alias old

Reply:
Yea sounds good